packages/test/test-service-load/src/FileLogger.ts

```diff
@@ -93,7 +93,19 @@ class FileLogger implements ITelemetryBufferedLogger {
 		) {
 			event.category = "generic";
 		}
-		this.baseLogger?.send({ ...event, hostName: pkgName, testVersion: pkgVersion });
+
+		const plainEvent: ITelemetryBaseEvent = {
+			...event,
+			hostName: pkgName,
+			testVersion: pkgVersion,
+		};
+		if (
+			process.env.FLUID_LOGGER_PROPS !== undefined &&
+			process.env.FLUID_LOGGER_PROPS.includes("displayName") === true
+		) {
+			plainEvent.details = JSON.stringify(process.env.FLUID_LOGGER_PROPS);
+		}
```

Comment:
Since it's already a string and we never JSON.parse()d it in this case, assigning it directly to `details` should be fine. And I think it's ok to put all of it in details, regardless of whether displayName is there or not.
```suggestion
		if (process.env.FLUID_LOGGER_PROPS !== undefined) {
			plainEvent.details = process.env.FLUID_LOGGER_PROPS;
		}
```